messages/index.js

```diff
@@ -701,6 +730,7 @@ module.exports = {
     fr: 'la propriété doit correspondre au format "{{=e.params.missingPattern}}"',
     id: 'harus memiliki pola pencocokan properti "{{=e.params.missingPattern}}"',
     it: 'deve avere un attributo che corrisponda al formato "{{=e.params.missingPattern}}"',
+    ka: 'ველი უნდა შეესაბამებოდეს შაბლონს "{{=e.params.missingPattern}}"',
```

Comment:
translation: "The field must match the template". The error should be that "the field that matches the pattern is required"

Reply:
Fixed